Makefile

```diff
@@ -1,3 +1,146 @@
+# VERSION defines the project version for the bundle. 
```

Comment:
Please add the command you ran to generate these files to the commit message. I am assuming rest of this commit need not be reviewed.